tests/fixtures_regress/tests.py

```diff
@@ -34,6 +36,17 @@
 _cur_dir = os.path.dirname(os.path.abspath(upath(__file__)))
 
 
+def is_ascii(s):
+    return all(ord(c) < 128 for c in s)
+
+
+def skipIfNonASCIIPath(test_func):
+    return unittest.skipIf(
+        not is_ascii(django.__file__) and six.PY2,
```

Comment:
No, your analysis is correct. In Python 2, sometimes `module.__file__` can contain absolute paths, but I don't remember the exact details and it's very unlikely.